tests/storybook.ts

```diff
@@ -0,0 +1,13 @@
+import { runInRepo } from '../utils.ts'
+import type { RunOptions } from '../types.d.ts'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'storybookjs/storybook',
+		branch: 'svelte-ecosystem-ci',
```

Comment:
```suggestion
		branch: 'next',
```

To be changed when https://github.com/storybookjs/storybook/pull/32564 is merged!